tsconfig.json

```
@@ -3,7 +3,7 @@
         "module": "commonjs",
         "esModuleInterop": true,
         "target": "es6",
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

We don't want to override this rule, otherwise, we can make a claim that we should have just used base javascript. 

There's the case for eslint being there to help us with code quality, but when it comes to typescript we want to make sure that it's as strict as possible. 
```suggestion
        "noImplicitAny": true,
```

This can be used for proof of concept while building something or testing things out, but proper types should be created at the time you'd like to push to dev/prod